src/xcompact3d.f90

```diff
@@ -273,11 +272,29 @@ subroutine init_xcompact3d()
   if (itype==2) then
      if(nrank.eq.0)then
         open(42,file='time_evol.dat',form='formatted')
+        if(mhd_active) then
+           open(43,file='mhd_time_evol.dat',form='formatted')
```

Comment:
Hard-coded IO unit number is not looking great ! But if it works just fine, ok.

Reply:
The hard-coded IO unit has always been there. I am ok to change it, but I would do it in a dedicated PR